src/components/layout/AddToWishlistButton/AddToWishlistButton.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useEffect } from 'react';
+import Button from '@mui/material/Button';
+import axios from 'axios';
+import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
+import { useNavigate } from 'react-router-dom';
+
+const AddToWishlistButton = ({ album }) => {
+  const [isAdded, setIsAdded] = useState(false);
+  const { status } = useAuth();
+  const navigate = useNavigate();
+  const isUserLoggedIn = status === AuthStatus.LoggedIn;
+
```

Comment:
Generally for all these `axios` calls; let's opt to use the base instance/client that is set up here: [https://github.com/Code-the-Dream-School/dd-prac-team4-front/blob/PS-108-Create-a-React-component-for-the-wishlist-button/src/apis/axiosClient.js](https://github.com/Code-the-Dream-School/dd-prac-team4-front/blob/PS-108-Create-a-React-component-for-the-wishlist-button/src/apis/axiosClient.js)

That way it already does `withCredentials` for us. And in the future if we want to make any other changes that should effect _all_ http calls from the frontend, there's a single place to make that change, insetad of having to edit every component.

Reply:
Done.